code/controllers/subsystems/mapping.dm

```diff
@@ -60,7 +60,7 @@ var/datum/controller/subsystem/mapping/SSmapping
 		if (banned_maps)
 			var/mappath = MT.mappath
 			if(list_find(banned_maps, mappath))
-				break
+				continue
```

Comment:
Can we move this thing to before we new the map_template datum? There's no point in newing it if it is banned.